### ISSUE TITLE
Error during Training

### ISSUE THREAD

**Issue by loginofdeath** [explosion/spaCy#2031]
I just wanted to test out how text classification works in spaCy and was trying out this code:

```
import spacy
nlp = spacy.load('en')

train_data = [
    (u"That was very bad", {"cats": {"POSITIVE": 0, "NEGATIVE": 1}}),
    (u"it is so bad", {"cats": {"POSITIVE": 0, "NEGATIVE": 1}}),
    (u"so terrible", {"cats": {"POSITIVE": 0, "NEGATIVE": 1}}),
    (u"I like it", {"cats": {"POSITIVE": 1, "NEGATIVE": 0}}),
    (u"It is very good.", {"cats": {"POSITIVE": 1, "NEGATIVE": 0}}),
    (u"That was great!", {"cats": {"POSITIVE": 1, "NEGATIVE": 0}})
]

textcat = nlp.create_pipe('textcat')
nlp.add_pipe(textcat, last=True)
textcat.add_label('POSITIVE')
textcat.add_label('NEGATIVE')
optimizer = nlp.begin_training()
for itn in range(10):
    for doc, gold in train_data:
        nlp.update([doc], [gold], sgd=optimizer)
doc = nlp(u'It is good.')
print(doc.cats)
doc = nlp(u'It is bad.')
print(doc.cats)
```
But its giving me these errors:
```
Traceback (most recent call last):
  File "/Users/loginofdeath/Documents/Artivatic/24Feb/huehue.py", line 17, in <module>
    optimizer = nlp.begin_training()
  File "/usr/local/lib/python2.7/site-packages/spacy/language.py", line 464, in begin_training
    sgd=self._optimizer)
  File "nn_parser.pyx", line 843, in spacy.syntax.nn_parser.Parser.begin_training
KeyError: u'token_vector_width'
```
Can someone help me?

* **Python version:** 2.7.14
* **Platform:** Darwin-16.4.0-x86_64-i386-64bit
* **spaCy version:** 2.0.6
* **Models:** en

**Comment by honnibal** [comment]
I think this I fixed this recently -- try updating to 2.0.9. If that doesn't work, try passing `token_vector_width=128` as an argument to `nlp.begin_training()`

I think there's another problem with your code though. If you do `nlp.load()` followed by `begin_training()`, you'll re-initialize the weights to all your models. I doubt this is what you want. You could create a blank `nlp` object with `spacy.blank()`. Alternatively you should be able to call `textcat.begin_training()` instead of `nlp.begin_training()`.

**Comment by loginofdeath** [comment]
Thank you, Matthew, for the lightning fast response. Adding the argument did not work. I had to update to the latest version of spaCy. Have a great day!
